app/components/TrackComponent/index.js

```diff
@@ -5,21 +5,27 @@ import { Card } from 'antd';
 import styled from 'styled-components';
 import T from '@components/T';
 import If from '@components/If';
+import history from '@app/utils/history';
 
 const TrackCard = styled(Card)`
   && {
     width: 22rem;
     height: auto;
     alignitems: start;
     flexwrap: wrap;
+    background-color: #231f20;
+    color: white;
+    font-weight: bolder;
```

Comment:
we had an interesting discovery a while back in one of our other projects. Safari refused to recognize anything other than absolute values for font-weight. Probably cause of the font-family that we used. However we've now formalised that we'll use distinct font-weights as far as possible. Please help me make this change